src/app/@theme/components/header/header.component.html

```diff
@@ -6,10 +6,12 @@
     <div class="logo" (click)="goToHome()">ngx-<span>admin</span></div>
   </div>
   <ngx-theme-switcher></ngx-theme-switcher>
+  
 </div>
 
 <div class="header-container">
-  <ngx-layout-direction-switcher></ngx-layout-direction-switcher>
+  <ngx-lang-switcher></ngx-lang-switcher>
+  <!-- <ngx-layout-direction-switcher></ngx-layout-direction-switcher> -->
```

Comment:
Why is this commented out?

Reply:
It's added by mistake :) I forgot to uncomment